lib/sdf-server/src/service/change_set/abandon_change_set.rs

```diff
@@ -41,6 +42,8 @@ pub async fn abandon_change_set(
         .await?
         .ok_or(ChangeSetError::ChangeSetNotFound)?;
 
+    let name = change_set.clone().name;
```

Comment:
nit: we can just call `change_set.name.to_owned()` so we only clone the field. Non-blocking and I'll fix in a follow-up.